index.bs

```diff
@@ -5008,7 +5015,7 @@ partial interface MLGraphBuilder {
 
 <details open algorithm>
   <summary>
-    To <dfn for="MLGraphBuilder" data-lt="reduce-op">create reduce operation</dfn> given [=string=] |op|, {{MLOperand}} |input|, {{MLReduceOptions}} |options|, and optional [=/list=] |allowedDataTypes|, run the following steps:
+    To <dfn for="MLGraphBuilder">create reduction operation</dfn> given [=string=] |op|, {{MLOperand}} |input|, {{MLReduceOptions}} |options|, and optional [=/list=] |allowedDataTypes|, run the following steps:
```

Comment:
👍 Yeah, "reduction" sounds more natural in full sentence context, as it matches the other noun-form "binary operation" and noun "pooling".

- The following reduce algorithms are supported.
- The following pooling algorithms are supported.
- The following argMin/argMax algorithms are supported.
- The element-wise binary operation algorithms ... as follows.
- The element-wise logical operation algorithms ... as follows.

(now, that does leave argMin/argMax feeling a little out of place from the rest, but "The following argMinization/argMaximization algorithms" sounds verbose :b).

(resolve me)